### PR TITLE
[CI] Downstream builds and setup concurrency

### PR DESCRIPTION
**Pull request by christophfroehlich**
Don't test them, if there is an error in this repo then the build will fail already.

### CODE REVIEW

**Comment by christophfroehlich** [comment]
@mergifyio backport humble

File: .github/workflows/rolling-pre-commit.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ on:
     branches:
       - master
 
+concurrency:
+  # cancel previous runs of the same workflow, except for pushes on master branch
```
Comment (fmauch) [inline .github/workflows/rolling-pre-commit.yml:10]: This doesn't contain the exception, right?

File: .github/workflows/rosdoc2.yml [hunk 2]
```diff
@@ -8,6 +8,10 @@ on:
       - control_msgs/rosdoc2.yaml
       - control_msgs/package.xml
 
+concurrency:
+  # cancel previous runs of the same workflow, except for pushes on master branch
```
Comment (fmauch) [inline .github/workflows/rosdoc2.yml:12]: This doesn't contain an exception for the master branch.

File: .github/workflows/rolling-pre-commit.yml (hunk 1 above)
Comment (christophfroehlich) [reply .github/workflows/rolling-pre-commit.yml:10]:
```suggestion
```

File: ros_controls.humble.repos [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    url: https://github.com/ros-controls/control_toolbox.git
+    version: ros2-master
+  ros-controls/kinematics_interface:
+    type: git
+    url: https://github.com/ros-controls/kinematics_interface.git
+    version: humble
+  ros-controls/realtime_tools:
+    type: git
+    url: https://github.com/ros-controls/realtime_tools.git
+    version: master
```
Comment (fmauch) [inline ros_controls.humble.repos:13]:
```suggestion
    version: humble
```

**Review by fmauch — APPROVED** [review]
Except from that little branch that looks fine

**Pull request merged**